### ISSUE TITLE
The Path of the Adept BE Paladin quest

### ISSUE THREAD

**Issue by Kaghol** [elysium-project/tbc-bug-tracker#565]
**Summary:** Unable to find Blood Knight Dawnstar’s corpse in Deatholme.

**Steps to Reproduce:** 

**Expected Results:** His corpse is supposed to be near the chained cauldron to receive a Blood Knight insignia.

**Actual Result:** No corpse found

**Additional Information:**

**Comment by Kaghol** [comment]
If this gets fixed, can you fix the door in SFK too so I can complete this chain!

**Comment by Kaghol** [comment]
I just noticed some quest items at the vendors in Org, would it be possible to do that for this paladin quest as well, so we can test the rest of the quest chain, as SFK is bugged.

**Comment by Kaghol** [comment]
This is still an issue after the launch, still can’t locate the Blood Knight Insignia in Deatholme.

**Comment by kolgarlol** [comment]
ya, quest still not works

**Comment by nbonfire** [comment]
> 
> 
> If this gets fixed, can you fix the door in SFK too so I can complete this chain!

Door in SFK works now FYI

**Comment by Kaghol** [comment]
Yes, SFK is fine, but still no body to get the Paladin insignia from. 😥
